drivers/aip31068/Makefile.dep

```diff
@@ -2,5 +2,7 @@ FEATURES_REQUIRED += periph_i2c
 
 USEMODULE += xtimer
 
-# necessary to fix driver initialization on esp32
-USEMODULE += esp_i2c_hw
+ifeq (esp32,$(CPU))
+	# necessary to fix driver initialization on esp32
+	USEMODULE += esp_i2c_hw
```

Comment:
```suggestion
  # necessary to fix driver initialization on esp32
  USEMODULE += esp_i2c_hw
```

Reply:
Replaced tabs with 2 spaces and squashed.